platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -257,106 +227,101 @@ public void testConstructorWithNullConfigurationFileExtension() {
                 null);
     }
 
-    @Test
-    public void testInitDoesNotCreateExistingProcessedAndFailedAndExportedDirectories()
-            throws IOException {
-        when(processedDirectory.exists()).thenReturn(true);
-        when(processedDirectory.isDirectory()).thenReturn(true);
-        when(failedDirectory.exists()).thenReturn(true);
-        when(failedDirectory.isDirectory()).thenReturn(true);
+    @Test(expected = IOException.class)
+    public void testInitDoesNotCreateExistingProcessedAndFailedDirectories() throws IOException {
 
-        ConfigurationAdminMigration configurationAdminMigration =
-                createConfigurationAdminMigratorWithNoFiles();
+        PowerMockito.doNothing()
+                .when(FileUtils.class);
+        FileUtils.forceMkdir(processedDirectoryPath.toFile());
+        PowerMockito.doThrow(new IOException())
+                .when(FileUtils.class);
+        FileUtils.forceMkdir(processedDirectoryPath.toFile());
+
+        PowerMockito.doNothing()
+                .when(FileUtils.class);
+        FileUtils.forceMkdir(failedDirectoryPath.toFile());
+        PowerMockito.doThrow(new IOException())
+                .when(FileUtils.class);
+        FileUtils.forceMkdir(failedDirectoryPath.toFile());
+
+        ConfigurationAdminMigration configurationAdminMigration = new ConfigurationAdminMigration(
+                configurationDirectoryStream,
+                processedDirectoryPath,
+                failedDirectoryPath,
+                configurationFileFactory,
+                configurationFilePoller,
+                configurationAdmin,
+                configurationFileExtension);
+
+        configurationAdminMigration.init();
         configurationAdminMigration.init();
 
-        verify(processedDirectory, never()).mkdir();
-        verify(failedDirectory, never()).mkdir();
         verify(configurationDirectoryStream).close();
+
     }
 
     @Test(expected = IOException.class)
     public void testInitFailsWhenProcessedDirectoryExistsButIsNotADirectory() throws IOException {
-        when(processedDirectory.exists()).thenReturn(true);
-        when(processedDirectory.isDirectory()).thenReturn(false);
-        when(failedDirectory.exists()).thenReturn(true);
-        when(failedDirectory.isDirectory()).thenReturn(true);
+        PowerMockito.doThrow(new IOException())
```

Comment:
Since we expect an exception and need to verify that the stream got closed, we need to use the `try/catch` pattern mentioned above. We can also verify that `FileUtils.forceMkdir(processedDirectoryPath.toFile());` was called.